src/rules/function-calc-no-unspaced-operator/__tests__/index.js

```diff
@@ -77,6 +77,12 @@ testRule(rule, {
     code: "a { top: calc(2rem + @fh+d*sf-as); }",
     description: "Less variable with symbols",
   }, {
+    code: "a { top: calc(100% - #{$foo}); }",
+    description: "Scss interpolation",
+  }, {
+    code: "a { top: calc(100% - #{map-get($container-max-widths, xl)}); }",
+    description: "Scss interpolation with function",
+  }, {
```

Comment:
Can we have a test showing that it errors with `a { top: calc(100%- #{$foo}); }` or `a { top: calc(100% -#{$foo}); }`?